src/Cake.Common.Tests/Unit/Tools/NUnit/NUnit3RunnerTests.cs

```diff
@@ -244,10 +253,14 @@ public void Should_Set_Commandline_Switches()
                         "--stoponerror \"--work=/Working/out\" \"--out=/Working/stdout.txt\" " +
                         "\"--err=/Working/stderr.txt\" --full " +
                         "\"--result=/Working/NewTestResult.xml;format=nunit2;transform=/Working/nunit2.xslt\" " +
```

Comment:
I haven't tried it in Cake - but NUnit itself currently crashes! https://github.com/nunit/nunit-console/issues/252 We'll be fixing that one - but it will be by returning an error message from the NUnit Console, rather than creating a feature!